src/scenes/components/select/select-showcase.component.tsx

```diff
@@ -4,7 +4,7 @@ import { Select, SelectElement, SelectProps } from '@ui-kitten/components';
 
 export const SelectShowcase = (props: SelectProps): SelectElement => {
 
-  const [selectedOption, setSelectedOption] = React.useState(props.selectedOption);
+  const [selectedOption, setSelectedOption] = React.useState(props.children);
```

Comment:
This example is not working at all

Reply:
[Select was fixed.](https://github.com/akveo/kittenTricks/pull/311/commits/3c80618895af175965ec5fe1c8d301f6779da406)